src/components/Viewer.tsx

```diff
@@ -43,7 +43,12 @@ function WrappedViewStateDeck(props: {
     const { width, height, maxZoom } = getLayerSize(firstLayerProps);
     const padding = deck.width < 400 ? 10 : deck.width < 600 ? 30 : 50; // Adjust depending on viewport width.
     const bounds = fitBounds([width, height], [deck.width, deck.height], maxZoom, padding);
-    setViewState(bounds);
+    // FIXME: For some reason deck.width & deck.height when there is no view
+    // state, resulting in a `NaN` zoom.
+    // This prevents us from setting the state until we have a valid state
+    if (!Number.isNaN(bounds.zoom)) {
+      setViewState(bounds);
+    }
```

Comment:
@will-moore Can't seem to track down why this is happening, really frustrating. I probably won't be have to have a closer look for a bit, but I wouldn't let it hold us back from iterating. i'll hvae to dig in later.